sql/sql_expression.cc

```diff
@@ -721,12 +720,12 @@ static Utf8String matchIdentifier(ML::Parse_Context & context,
                 }
                 else if (context.match_literal('"')) {
                     token.ignore();
-                    return result;
+                    return true;
```

Comment:
After discussion, the handling of the expression above is different and this is actually part of the new requirements.